BlueprintUI/Tests/RenderPassCacheTests.swift

```diff
@@ -3,7 +3,7 @@ import XCTest
 
 final class RenderPassCacheTests: XCTestCase {
     func test_caching() {
```

Comment:
Should we add a test that validates the rounding logic is cached appropriately?

Reply:
Probably a good idea! Will do.